flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-mysql/src/test/java/org/apache/flink/cdc/connectors/mysql/source/MySqlPipelineITCase.java

```diff
@@ -501,6 +501,22 @@ public void testSchemaChangeEvents() throws Exception {
             expected.add(
                     new DropTableEvent(
                             TableId.tableId(inventoryDatabase.getDatabaseName(), "customers")));
+
+            // Test create table DDL
+            statement.execute(
+                    String.format(
+                            "CREATE TABLE `%s`.`newlyAddedTable1`(id int, id2 int, primary key(id));",
```

Comment:
Could you cover more data types like our full types test?

Reply:
Done.